package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "JS-restaurant-page",
+  "version": "1.0.0",
+  "description": "> This is a library Web-App made with Javascript and Bootstrap 5 where the user can add information about books, delete created books, and edit the status if the user has already read it.",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
+    "build": "webpack",
+    "start": "webpack serve --open"

```

Comment:
- [ ] Kindly specify the script mode to either development or production for example:
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```